amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -264,7 +207,7 @@ export function* urlDidUpdateWorker(action: UrlDidUpdateRequest): SagaIterator {
 
   const state = yield select(getSearchState);
   if (!!term && state.search_term !== term) {
-    yield put(searchAll(term, resource, parsedIndex));
+    yield put(searchAll("external_page_view", term, resource, parsedIndex));
```

Comment:
Not necessarily an `external_page_view`, but a browser navigation. Browser forward and backward, external URL, breadcrumbs (maybe?).

Reply:
Considering `loadFromUrl` for this type